package.json

```diff
@@ -92,6 +92,24 @@
             "items": "string",
             "default": [],
             "markdownDescription": "mode test"
+          },
+          "junbae-mode.location": {
+            "type": "string",
+            "default": "top",
+            "enum": [
+              "top",
+              "bottom"
+            ],
+            "markdownDescription": "Junbae is located on the screen"
+          },
+          "junbae-mode.gaugeColor": {
+            "type": "string",
+            "default": "white",
+            "enum": [
+              "white",
+              "red"
+            ],
+            "markdownDescription": "gauge color change"
```

Comment:
Could you order this properties?
I think `mode - location - gaugeColor` is the best (in order of importance)

Reply:
I see